src/components/MissingFacilitiesModal.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal"> 
+          <ion-icon :icon="close" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ $t("Missing facilities") }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content>
+
+    <ion-item v-for="(items, facilityId) in itemsByFacilityId" :key="facilityId" lines="full">
+      <ion-label>
+        {{ facilityId }}
+        <p>{{ items?.length }} {{ $t("line items") }}</p>
+      </ion-label>
+      <ion-select interface="popover" slot="end" :placeholder="$t('Map facility')" @ionChange="updateFacility($event, facilityId)">
+        <ion-select-option v-for="facility in facilities" :key="facility.facilityId" :value="facility.facilityId">{{ facility.facilityName }}</ion-select-option>
+      </ion-select>
+    </ion-item>
+
+    <ion-fab vertical="bottom" horizontal="end" slot="fixed">
+      <ion-fab-button @click="save">
+        <ion-icon :icon="saveOutline" />
+      </ion-fab-button>
+    </ion-fab>
+
+  </ion-content>
+</template>
+<script lang="ts">
+import { 
+  IonButtons,
+  IonButton,
+  IonContent,
+  IonFab,
+  IonFabButton,
+  IonHeader,
+  IonItem,
+  IonIcon,
+  IonLabel,
+  IonSelect,
+  IonSelectOption,
+  IonTitle,
+  IonToolbar,
+  modalController,
+} from "@ionic/vue";
+import { defineComponent } from "vue";
+import { close, saveOutline } from "ionicons/icons";
+import { useStore } from "@/store";
+import { mapGetters } from "vuex";
+import { showToast } from "@/utils";
+import { translate } from "@/i18n";
+export default defineComponent({
+  name: "MissingFacilitiesModal",
+  components: {
+    IonButtons,
+    IonButton,
+    IonContent,
+    IonFab,
+    IonFabButton,
+    IonHeader,
+    IonIcon,
+    IonItem,
+    IonLabel,
+    IonSelect,
+    IonSelectOption,
+    IonTitle,
+    IonToolbar 
+  },
+  props: ["itemsWithMissingFacility", "facilities"],
+  data() {
+    return {
+      itemsByFacilityId: {},
+      mapFacility: {} as any
```

Comment:
name mapFacility represents an action
```suggestion
      facilityMapping: {} as any
```